app/containers/UnlockEtherContainer.tsx

```diff
@@ -117,40 +157,57 @@ class UnlockEtherContainer extends React.Component<IUnlockEtherContainerProps> {
       <div className={styles.section}>
         {willBePenalized
```

Comment:
This is a good catch. In general we cannot assume everyone is penalized (page may live more than 18 months). So we should check this but differently, see below